### PR TITLE
Add variable for tags

### PR DESCRIPTION
**Pull request by lonegunmanb**
## Issue number

#35 

## Checklist before requesting a review
- [ ] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [ ] I have executed pre-commit on my machine
- [ ] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -6,10 +6,10 @@ resource "random_integer" "this" {
 locals {
   account_name          = coalesce(var.account_name, "azure-openai-${random_integer.this.result}")
   custom_subdomain_name = coalesce(var.custom_subdomain_name, "azure-openai-${random_integer.this.result}")
-  tags = var.default_tags_enabled ? {
+  tags = merge(var.default_tags_enabled ? {
```
Comment (zioproto) [inline main.tf:9]: @soferreira do we want to give the possibility to add additional tags also when the default tags are enabled ? Or this version of the logic works ?

@JinLee794 feedback is welcome

Comment (zioproto) [reply main.tf:9]: Ah wait, I misunderstood the code.

I understand now correctly that `var.tags` is always merged in the tags, regarless of the value of `var.default_tags_enabled`. 

Looks good to me !

**Review by zioproto — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (soferreira) [reply main.tf:9]: Looks to me!

**Review by soferreira — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (JinLee794) [reply main.tf:9]: Looks good to me!

**Pull request merged**